pkg/fsutils/basepath.go

```diff
@@ -25,8 +25,7 @@ func AllRelativePathModes() []string {
 
 func GetBasePath(ctx context.Context, mode, cfgDir string) (string, error) {
 	if mode == "" {
-		// TODO(ldez): v2 the default should be cfg or gomod.
-		mode = RelativePathModeWd
+		mode = RelativePathModeCfg
 	}
```

Comment:
Nit:

Not sure if the following is more readable, but it can be written as:

```go
func GetBasePath(ctx context.Context, mode, cfgDir string) (string, error) {
	switch mode {
	case "", RelativePathModeCfg:
```

or

```go
func GetBasePath(ctx context.Context, mode, cfgDir string) (string, error) {
	mode = cmp.Or(mode, RelativePathModeCfg)

	switch mode {
	case RelativePathModeCfg:
```

Reply:
If the variable `mode` is ever used later in the scope it's prone to bugs since it's not assigned to the expected value. I guess `cmp.Or()` is the new way to use syntactic sugar for what the current code does already

Reply:
One-liners are not better than explicit `if`: we write code for humans, not computers, number of LOC is not a valid metric for compiled language.

IMO, the usage of the package `cmp` outside of the initial intended scope of comparison is not a recommendable practice.

I will use `cmp` but remember that my philosophy is always readability over LOC.
The code is not a candy, it doesn't need a lot of sugar.